_component_design/choice-tiles.md

```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
 layout: component
 category: Components
 usage: >
-  Choice tiles are an alternative selector pattern to radio buttons and checkboxes that provide a means of both displaying choices more visually and of providing a more detailed description of a choice.
+  A choice tile represents a selection that a user can make on a page. As an alternative to a radio button or checkbox, a choice tile contains an icon that graphically represents the choice and a description that provides more detail about the choice.
```

Comment:
choice tile is a selectable object/component. Iconography is optional.

remove stuff about icon. selectable component that you can use a more graphic alternative to a radio and checkbox.

Reply:
Done.